### DISCUSSION TITLE
Data import

### DISCUSSION THREAD

**Discussion by mtnbiker777** [photoprism/photoprism#993] [General]
I'm currently using photoshow and have spent a lot of time adding descriptions to all of my pictures. Is there a way to import that data into PhotoPrism. The description is stored in an XML file with the original filename. I can script putting the XML and image in the same directory if need be. Ideally I would like this description to end up either/or both the description of the photo or as the label for the image. I can see that the manually entered description ends up in a sidecar yml file ... And yes, I'm new so be gentle please.

**Comment by lastzero** [comment]
Reading XML files possible, but requires work - so not something we can deliver short-term. Ideally the XML files should be named the same as the original file so that we don't need to search it first. You're welcome to create a feature request issue for this.
